CommandLine/libEGM/gmx.cpp

```diff
@@ -430,6 +432,7 @@ void PackRes(std::string &dir, int id, pugi::xml_node &node, google::protobuf::M
               if (isFilePath) {  // again gotta prepend the gmx's path & fix the string to be posix compatible
                 value = GMXPath2FilePath(dir, value);
               }
+              if (!refType.empty() && value == "<undefined>") break;
```

Comment:
I'm not sure what the `!refType.empty()` check does for you. Might be good to document that line.

Reply:
You do know that code strings go through this branch right? If you don't check `!refType.empty()` then you don't that it's a resource reference you are comparing. Meaning you couldn't create a script that contained `"<undefined>"`, not that you'd want to do that anyway.